fluentd/lib/fluent-plugin-collected/lib/fluent/plugin/in_collected_tail_monitor.rb

```diff
@@ -9,28 +9,15 @@ class CollectedTailMonitorInput < Fluent::Plugin::Input
     include Fluent::Plugin::PrometheusLabelParser
 
     helpers :timer
-
-    #ideally interval must be rotatewait > 0 ? rotatewait/2 : 1.0
-    #and if interval >= rotate_wait we should log a warning.
-    #For rotatewait = 5 sec, interval can be set to 2 sec
-
-
     config_param :interval, :time, default: 2
     attr_reader :registry
 
-    MONITOR_IVARS = [
-      :tails,
-    ]
-
-    REGEX_VAR_LOG_PODS = Regexp.new('/var/log/pods/(?<namespace>[a-z0-9-]+)_(?<pod_name>[a-z0-9-]+)_(?<pod_uuid>[a-z0-9-]+)/(?<container_name>[a-z0-9-]+)/.*\.log$')
-    REGEX_VAR_LOG_CONTAINERS = Regexp.new('/var/log/containers/(?<pod_name>[a-z0-9-]+)_(?<namespace>[a-z0-9-]+)_(?<container_name>[a-z0-9-]+)-(?<docker_id>[a-z0-9]{64})\.log$')
-    REGEX_LOG_PATH=/(#{REGEX_VAR_LOG_PODS})|(#{REGEX_VAR_LOG_CONTAINERS})/
+    MONITOR_IVARS = [:tails].freeze
+    REGEX_LOG_PATH = Regexp.new('.*/(?<namespace>[a-z0-9-]+)_(?<pod_name>[a-z0-9-]+)_(?<pod_uuid>[a-z0-9-]+)/(?<container_name>[a-z0-9-]+)/.*\.log$').freeze
```

Comment:
This blocks us from possibly pushing this into older releases.  Are you sure we will never push this version of the image into logging releases older then 5.4

Reply:
Do you mean you want to re-enable support for /var/log/containers?

Reply:
I'm suggesting that if we do not support /var/log/containers then this blocks us from taking fluent 1.14.5 into previous releases of logging that are still using /var/log/containers.  If we truly have no intentions of moving 1.14.5 into anything earlier then 5.4 then no need to alter what is happening here

Reply:
Understood, I will restore support for /var/log/containers. I wanted to simplify but did not think that through.